docker/jdk9/x86_64/ubuntu/Dockerfile

```diff
@@ -40,8 +40,18 @@ RUN apt-get update \
     git \
     unzip \
     realpath \
+    g++-4.8 \
+    gcc-4.8 \
 && rm -rf /var/lib/apt/lists/*
 
+# Make sure build uses 4.8
+RUN rm /usr/bin/gcc \
```

Comment:
Isn't this forcing all builds to use the older gcc?

Reply:
Yeah, I'm inclined to agree with @karianna's implication that this is not helpful to all builds, so should be kept separate.

I can imagine (but have no evidence) that down-leveling the compiler will hurt performance for code such as Hotspot, and our builds should be optimal for the included components.

Reply:
In #166 I was specifically asked to move this code into the base Docker image so I also did that here. Note that OpenJDK8 documents gcc-4.8 (granted not on Ubuntu 16.04) as the build tool chain here https://wiki.openjdk.java.net/display/Build/Supported+Build+Platforms .